### ISSUE TITLE
lib markdown code issues

### ISSUE THREAD

**Issue by lwasser** [earthlab/earthlab.github.io#44]
Hey @mbjoseph there's an issue with the libs r code:

All of the lib files are being written with BOOLEANS in quotes. This actually negates some of the boolean properties in jekyll. I have a look at the code. How do we force R to not include quotes for these lines? example file: https://github.com/earthlab/earthlab.github.io/blob/master/org/lang-lib/libs/JSAnimation.md

i found this code but am not sure how to remove the quotes. unless it's as easy as deleting the double quotes?

```
    yaml <- list(layout = "post-by-category",
                 category = "tutorials",
                 title = paste(df$libs, "-", firstup(df$lang),
                               "Data Intensive Tutorials"),
                 permalink = paste0("/tutorials/", 
                                    df$lang, "/", df$libs),
                 comments = "false",
                 author_profile = "false",
                 language = df$lang,
                 library = df$libs,
                 langSide = "true") %>%
```

Can you please fix this or tell me how to?

**Comment by lwasser** [comment]
Also title should be in quotes.

**Comment by lwasser** [comment]
here is an example of a page that is messed up on the live site. 

https://earthlab.github.io/tutorials/software/r/scatterplot3d

**Comment by mbjoseph** [comment]
Hmm weird. Is it sufficient to have `yes` or `no` instead of `true` and `false`? That seems to be the default behavior in the `yaml` R package when using R's native boolean operators in those lists.

**Comment by lwasser** [comment]
I don’t know. I’ve always used true or false. I code statements like {% if page.comments %} do something if it’s true {% endif %}

Anything in quotes gets red as a string. [] = array and true / false is Boolean.

I’ve never tried yes / no as Booleans. Does R accept yes/no as Booleans?

**Comment by mbjoseph** [comment]
Ok - maybe liquid accepts yes and no as true and false. It seems weird that the R yaml package translates TRUE to yes otherwise....

**Comment by lwasser** [comment]
ok. yea i've only used true and false.

also - can you please have a look at this: 

https://earthlab.github.io/tutorials/software/python/JSAnimation

i cant find a tutorial that uses this library. do you know of one? if not i will remove it from the yaml list.

**Comment by mbjoseph** [comment]
I suspect that file was autogenerated, and then I removed the tutorial that it was used in (because it didn't render anyway). I opened #45 to address this - good catch!

**Comment by mbjoseph** [comment]
Solved by bb81c6f47ee3c6312216de4dbe2b75e6510de355
